l10n_it_rea/models/res_company.py

```diff
@@ -0,0 +1,54 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+from odoo.tools.misc import formatLang
+
+
+class Company(models.Model):
+    _inherit = 'res.company'
+
+    rea_office = fields.Many2one(
+        'res.country.state', string='Office Province',
+        related='partner_id.rea_office')
+    rea_code = fields.Char('REA Code', size=20, related='partner_id.rea_code')
+    rea_capital = fields.Float('Capital', related='partner_id.rea_capital')
+    rea_member_type = fields.Selection(
+        [('SU', 'Unique Member'),
+         ('SM', 'Multiple Members')], 'Member Type',
+        related='partner_id.rea_member_type')
+    rea_liquidation_state = fields.Selection(
+        [('LS', 'In liquidation'),
+         ('LN', 'Not in liquidation')], 'Liquidation State',
+        related='partner_id.rea_liquidation_state')
+
+    @api.onchange(
```

Comment:
Changing the *related* data in the partner does not recompute the `company_registry`, here the steps:
1. Fill the data in the company: the `company_registry` is correctly filled.
2. Go to the partner bound to the company and change a field.
3. Go back to the company: `company_registry` is incorrect (it still has the old computed value)

Maybe a computed field would fix this?

Reply:
@SimoRubi it can't be a computed field because it must be freely editable for possible non Italian companies

Reply:
Then maybe the behavior I described should be included in the 'known bugs`